### PR TITLE
Add Uberalls client tests

### PR DESCRIPTION
**Pull request by sectioneight**

### CODE REVIEW

File: src/main/java/com/uber/jenkins/phabricator/uberalls/UberallsClient.java [hunk 1]
```diff
@@ -67,13 +66,12 @@ public String getBaseURL() {
         return this.baseURL;
     }
 
-    public CodeCoverageMetrics getParentCoverage(Differential differential) {
```
Comment (sectioneight) [inline src/main/java/com/uber/jenkins/phabricator/uberalls/UberallsClient.java]: minor refactor to remove dependency on Differential class when all we need is the SHA of the parent commit

**Comment by sectioneight** [comment]
@jjx to review please

**Comment by sectioneight** [comment]
Going to self-merge, this is pretty innocuous.

File: src/main/java/com/uber/jenkins/phabricator/uberalls/UberallsClient.java [hunk 2]
```diff
@@ -67,13 +66,12 @@ public String getBaseURL() {
...
+    public CodeCoverageMetrics getParentCoverage(String sha) {
+        if (sha == null) {
```
Comment (jjx) [inline src/main/java/com/uber/jenkins/phabricator/uberalls/UberallsClient.java]: Use isBlank? We should add Apache StringUtils as a dependency at some point.

Comment (sectioneight) [inline src/main/java/com/uber/jenkins/phabricator/uberalls/UberallsClient.java]: Yup good idea. Was just preserving existing behavior, but you're right that that's more correct. Will fix in a future PR

**Pull request merged**